src/services/user/services/ForcePasswordChange.js

```diff
@@ -0,0 +1,83 @@
+const { BadRequest } = require('@feathersjs/errors');
+const { disallow } = require('feathers-hooks-common');
+const { authenticate } = require('@feathersjs/authentication');
+const { passwordsMatch } = require('../../../utils/passwordHelpers');
+const bcrypt = require('bcryptjs');
```

Comment:
Codacy found an issue: [`bcryptjs` import should occur before import of `../../../utils/passwordHelpers`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)